pyproject.toml

```diff
@@ -112,6 +112,7 @@ pytest-xdist = "^2.5.0"
 pylint = "^2.13.9"
 Markdown = "3.3.6"
 jupyter = "^1.0.0"
+moviepy = "^1.0.3"
```

Comment:
Eventually we should make these optional dependencies, because otherwise installing layer will take a long time due to unneeded dependencies.

Reply:
It's a dev dependency

Reply:
👍